src/components/filters/MeasureValueFilter/OperatorDropdownItem.tsx

```diff
@@ -5,12 +5,16 @@ import classNames from "classnames";
 import noop = require("lodash/noop");
 import capitalize = require("lodash/capitalize");
 import { string as stringUtils } from "@gooddata/js-utils";
+import BubbleHoverTrigger from "@gooddata/goodstrap/lib/Bubble/BubbleHoverTrigger";
+import Bubble from "@gooddata/goodstrap/lib/Bubble/Bubble";
 
 import { getOperatorTranslationKey, getOperatorIcon } from "../../../helpers/measureValueFilterOperator";
 
 export interface IOperatorDropdownItemOwnProps {
     selectedOperator: string;
     operator: string;
+    showBuble?: boolean;
```

Comment:
Maybe we could skip the `showBubble` property and just turn it on/off by setting `bubbleText` property?

Reply:
ok, the only bubbleText property is used now